Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM samdoshi/haskell-stack:latest
+
+RUN apt-get update
+RUN apt-get install -y mecab libmecab-dev mecab-ipadic-utf8
```

Comment:
These two commands could be concatenated with `&&`, and should be so, I suppose.